libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/LoginOptionsActivity.kt

```diff
@@ -215,6 +227,33 @@ fun BootConfigView() {
                 cursorColor = colorScheme.tertiary,
             ),
         )
+
+        Button(
+            modifier = Modifier.padding(PADDING_SIZE.dp).fillMaxWidth(),
+            shape = RoundedCornerShape(CORNER_RADIUS.dp),
+            contentPadding = PaddingValues(PADDING_SIZE.dp),
+            colors = ButtonColors(
+                containerColor = colorScheme.tertiary,
+                contentColor = colorScheme.tertiary,
+                disabledContainerColor = colorScheme.surfaceVariant,
+                disabledContentColor = colorScheme.surfaceVariant,
+            ),
+            enabled = validInput,
+            onClick = {
```

Comment:
On iOS, the view does not interact with SalesforceSDKManager (or anything else) directly. Instead it is passed a `onUseConfig` at initialization and uses it when the button is clicked. See [here](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/01a3ac3ffbc0c1e144cbd370e750fa4e7d715e84/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/DevConfig/BootConfigEditor.swift#L117).